server/src/main/java/org/opensearch/indices/NodeIndicesStats.java

```diff
@@ -276,6 +351,14 @@ public List<IndexShardStats> getShardStats(Index index) {
         }
     }
 
+    public CommonStats getIndexStats(Index index) {
```

Comment:
Can you please rewrite this properly? Also, do we really need it here if it is only used in tests? I think you can a new class with extend `NodeIndicesStats`.